prow/tide/codereview.go

```diff
@@ -161,3 +169,137 @@ func CodeReviewCommonFromPullRequest(pr *PullRequest) *CodeReviewCommon {
 
 	return crc
 }
+
+type CodeReviewEntity interface {
+	PullRequest
+}
+
+// CodeReviewInteractor is the interface implemented by each source code
+// providers, such as GitHub and Gerrit.
+type CodeReviewInteractor interface {
```

Comment:
Why not simply `Provider`?

Similiar comment for the fieldname in the `Tide` struct, it is currently named `cri`, that is super cryptig (and my first association with that is `ContainerRuntimeInerface`)

Reply:
done, I like the new name